plugins/bcc-login/includes/class-bcc-notifications.php

```diff
@@ -14,9 +14,14 @@ function __construct( BCC_Login_Settings $settings, BCC_Coreapi_Client $core_api
 
     }
 
-    // NB: This does not run if a post is published without first being saved as a draft.
+    //wpml plugin
+    $active_plugins = get_option('active_plugins');
+    $wpml_plugin = 'sitepress-multilingual-cms/sitepress.php';
+
+
+    // NB: This does not run if a post is published without first being saved as a draft, and the wpml plugin is installed
     public function on_post_status_transition(  $new_status, $old_status, $post ) {
-       if ('publish' === $new_status && 'publish' !== $old_status && in_array( $post->post_type, $this->settings->notification_post_types )) {
+       if ('publish' === $new_status && 'publish' !== $old_status && in_array( $post->post_type, $this->settings->notification_post_types ) && in_array($wpml_plugin, $active_plugins)) {
```

Comment:
The check for WPML shouldn't be here, since the plugin is designed to work without WPML as well.

Currently we already use the following check to test if WPML is installed and active (further down in the code):
`defined('ICL_SITEPRESS_VERSION')`

This check could be assigned to a variable, and then you can check that variable before using the wpml functions to get the current language, switch language etc.